GOVERNANCE.md

```diff
@@ -0,0 +1,79 @@
+<!--
+SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
+SPDX-FileContributor: Stephan Druskat
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+## Governance
+
+The HERMES project employs a lightweight governance model.
+
+All decisions are made by the [HERMES Steering Group](#hermes-steering-group) 
+together with the maintainer(s) of the repositories affected by the decision.
+
+Decisions that adhere to a single [repository in the scope of the HERMES project](#hermes-project-repositories)
+are made by *lazy consensus*: 
+
+1. The maintainer of the repository makes a decision and notifies the HERMES Steering Group
+(using `@softwarepub/hermes-steering-group` mention).
+2. If no members of the Steering Group veto the decision within the decision period of one week (168 hours),
+the decision becomes effective and can be acted upon. 
+The maintainer and any member of the Steering Group can ask for a longer decision period, 
+which must be granted. 

```

Comment:
As this could lead to an infinite block by a block by a single member, maybe we should add a fallback here:

```suggestion
The maintainer and any member of the Steering Group can ask for a longer decision period, 
which must be granted. 
During this period, the steering committee should meet to discuss the decision.
```